.github/workflows/merge-bump-packages-pr.yaml

```diff
@@ -3,18 +3,23 @@ on:
   workflow_dispatch:
   schedule:
     # Each Tuesday at 5 AM UTC
-    - cron: '0 5 * * 2'
+    - cron: "0 5 * * 2"
 
 jobs:
   merge_bump_packages_pr:
     name: Merge bump packages PR
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - name: Create Github App Token
+        uses: actions/create-github-app-token@v1
```

Comment:
Other create github app token actions are using `mongodb-js/devtools-shared/actions/setup-bot-token@ni/github-app-action`, is there a reason why this one is an exception? Should we document it?

Reply:
This one doesn't actually make commits/push. It's fairly inconsequential though and if you think it'll be less confusing we can switch it to `setup-bot-token`.

Reply:
Yes, I think it's confusing that actions with exactly the same names are using different actions, so it would be very helpful to either add a comment why or just use the same everywhere if it doesn't make a difference, maybe make it configurable to cover both cases if we don't want to use existing actions and want to have our own. New actions are often created by just using existing ones as templates, I'm not sure how one would decide which version of "Create Github App Token" to use without a clarification